client/src/components/Ticketing/ticketingmanager/Events/deleteConfirm_depreciated.tsx

```diff
@@ -4,27 +4,27 @@ import React, {CSSProperties} from 'react';
  * @property {object} popUpContainer - A property container for pop up
  */
 const popUpContainer: CSSProperties = {
-  'position': 'fixed',
-  'background': '#00000050',
-  'width': '100%',
-  'height': '100vh',
-  'top': '0',
-  'left': '0',
+  position: 'fixed',
```

Comment:
Any reason why this file wasn’t deleted as well?

Reply:
Hmm when I removed it initially, there were dependency errors but now when I check, it seems to have been resolved. Not sure why, but I'll remove it